web/src/markdown.js

```diff
@@ -398,7 +398,7 @@ function handleUnicodeEmoji({unicode_emoji, get_emoji_name}) {
 
     // This unqualifies qualified emoji, which helps us make sure we
     // can match both versions.
-    unqualified_unicode_emoji = unicode_emoji.replace(/\uFE0F/, "");
+    const unqualified_unicode_emoji = unicode_emoji.replace(/\uFE0F/, "");
```

Comment:
This fix should be moved to the “emoji: Match emoji sequences in markdown” commit.

Reply:
🙈 thanks